azure-devops/azext_devops/dev/common/services.py

```diff
@@ -355,6 +355,10 @@ def resolve_instance_project_and_repo(
         _raise_team_project_arg_error()
     if repo_required and repo is None:
         _raise_repo_requird_arg_error()
+
+    if not check_organization_in_azure(organization):
+        logger.warning("Support for Azure DevOps Server, is under development")
```

Comment:
@aaronhallberg 
need your input on this string

Reply:
I think we should be more explicit here. Something like "_The Azure DevOps Extension for the Azure CLI does not support Azure DevOps Server._"

Reply:
@roshan-sy : Let's take this change and we will be good to merge this PR

Reply:
Done